db/migrations/20230605221226_notification_attributes_migration.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class NotificationAttributesMigration extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up(): void
+    {
+        // add new columns
+        $this->execute("
+            ALTER TABLE `notifications`
+                ADD COLUMN `attributes` VARCHAR(200) NOT NULL DEFAULT '[]'
```

Comment:
```suggestion
                ADD COLUMN `attributes` VARCHAR(2000) NOT NULL DEFAULT '[]'
```
Let's make this much larger than we think we need, running into "I stored too much JSON and the DB is truncating it" can be hard to figure out sometimes